x-pack/plugins/security_solution/common/test/ess_roles.json

```diff
@@ -132,5 +132,22 @@
         "base": []
       }
     ]
+  },
```

Comment:
I think you could also use the existing `reader` role?

Reply:
I gave it a go but unfortunately `reader` has read privileges for the `risk-score.risk-score-*` index so doesn't allow me to test the full callout.